### DISCUSSION TITLE
Prefil an entry with a fixed template

### DISCUSSION THREAD

**Discussion by fvanhove** [statamic/cms#8413] [General]
Hey peeps 

Trying to simulate  a specific case, not sure if all existing modules support this behaviour 😄 

Like a good lad, we have pages that make use of a Blueprint.  This is build in a way that users can click together their own website using the replicator and Fieldset sections. 

What we want to provide the content creator is the following: 
when creating a new entry we want to offer them the option to start from a black template
when creating a new entry we want to offer them the option to start from a prefilled template (e.g. landing page 1)

  Is this an option, or should we be looking at building an addon that does this?

**Comment by jasonvarga** [comment]
When you say template, do you mean the front-end view? Or do you mean the entry should have some pre-filled data?

If it's the data, you'll probably need to reach for an addon.

**Reply by fvanhove** [reply]
Hey @jasonvarga  

Thanks for the answer, 
I meant the Control panel view where the content creator, can quickly scaffold page entries using existing presets. 
(Not to be confused with changing blueprints). 

We build the systems to work section based. e.g: Hero section, Image section, cta section, ... 
And ofcourse certain compositions work best together. 

So the goal was for the content creator to either start a blank entry or select a "templates/example/scaffold" they can use whenever they create a new "page entry". The latter would make sure they get the best possible result. ( ofcourse we would define these "templates" during the development process )

With the extra context provided, i'm still assuming this is what you reffer to as: "the data" and I'd have to build an addon?

**Reply by jasonvarga** [reply]
I guess you could also add `default` values to all the fields in the blueprint.

**Reply by fvanhove** [reply]
Aah yes, altho that would only allow me to have 1 "template". And I don't know how efficient this would be with the repeater fieldType. 

But I can already dilude from the conversation that it's not a feature that i'm overlooking and I most likely have to make an addon.  :-) 
Thanks!

**Comment by jasonvarga** [comment]
Another naive solution could be to have a draft entry of each type sitting there, and use the "duplicate" action to start creating a new entry instead of the "create entry" button.

**Reply by fvanhove** [reply]
Yea, that actually crossed my mind aswell. The end result is exacly what I am aiming for.  

I will most likely use this approach in one or two sites to see the initial reactions of the content creators and if needed I will still adapt to an addon that has the same end result but a more dedicated ux flow.
